index.js

```diff
@@ -79,7 +79,7 @@ var Sqlite3Driver = Base.extend({
       constraint.push('UNIQUE');
     }
 
-    if (spec.defaultValue) {
+    if (spec.defaultValue !== undefined) {
```

Comment:
This wouldn't catch a empty string though.

```javascript
var test = "";

if ( test !== undefined && test !== null )
        console.log('I passed!');
```

I think it is more viable to check for the values that are really allowed to be 0 instead of checking for all that are not.